spec/markdown-preview-view-spec.coffee

```diff
@@ -99,6 +99,17 @@ describe "MarkdownPreviewView", ->
           }
         """
 
+  describe "editor behaves read-only", ->
+    beforeEach ->
+      waitsForPromise ->
+        preview.renderMarkdown()
+
+    describe "when a code block is rendered", ->
+      it "it does not have line-cursor decorations", ->
```

Comment:
I would nest this example under `describe "code block conversion to atom-text-editor tags"`, strip the containing `describe` calls, and change the description to `it "removes line decorations on rendered code blocks"`. In general, I try to structure specs as follows:

* describe - a noun or a situation
  * it - a statement about state or how an operation changes state

It's like a conversation...

Person 1: "Describe a dog"
Person 2: "It barks"
Person 1: "Describe a dog, when the dog is hungry" (a nested describe)
Person 2: "It begs for food"